src/components/badges/subcomponents/Badge.spec.jsx

```diff
@@ -60,14 +60,4 @@ describe('<Badge />', () => {
 
     expect(badge.hasClass('sg-badge--rounded')).toEqual(true);
   });
-
-  it('logs error when no child has been passed', () => {
```

Comment:
why removing those checks?

Reply:
this error was thrown in runtime by `prop-types` before, right now it's a dev-time error in flow :)

Reply:
👍